comms/EclCC.cpp

```diff
@@ -175,6 +175,11 @@ class CEclCC : public IEclCC, public clib::CLockableUnknown
         return false;
     }
 
+    const TCHAR * GetToolTip(CString key, std::_tstring & tooltip)
```

Comment:
change to:  `const std::_tstring & key`

Note:  In general its preferable to use "std::_tstring" in preference to CString (and limit CString usage to marshaling between BSTR and VARIANTs) - but I do appreciate there quite a few places that use CString still.

Reply:
@dehilsterlexis bump